[2023-07-22T05:54:10.614+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table scheduled__2023-07-13T00:00:00+00:00 [queued]>
[2023-07-22T05:54:10.832+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table scheduled__2023-07-13T00:00:00+00:00 [queued]>
[2023-07-22T05:54:10.835+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-07-22T05:54:10.840+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-07-22T05:54:10.841+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-07-22T05:54:10.989+0000] {taskinstance.py:1383} INFO - Executing <Task(PostgresOperator): create_table> on 2023-07-13 00:00:00+00:00
[2023-07-22T05:54:11.009+0000] {standard_task_runner.py:55} INFO - Started process 5148 to run task
[2023-07-22T05:54:11.109+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'create_table', 'scheduled__2023-07-13T00:00:00+00:00', '--job-id', '36', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmpey3e5yyv']
[2023-07-22T05:54:11.148+0000] {standard_task_runner.py:83} INFO - Job 36: Subtask create_table
[2023-07-22T05:54:11.773+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing.create_table scheduled__2023-07-13T00:00:00+00:00 [running]> on host ef71d3f52879
[2023-07-22T05:54:12.093+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2023-07-13T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-07-13T00:00:00+00:00
[2023-07-22T05:54:12.125+0000] {base.py:71} INFO - Using connection ID 'postgres' for task execution.
[2023-07-22T05:54:12.149+0000] {sql.py:315} INFO - Running statement: 
            CREATE TABLE IF NOT EXISTS users (
                firstname TEXT NOT NULL,
                lastname TEXT NOT NULL,
                country TEXT NOT NULL,
                username TEXT NOT NULL,
                password TEXT NOT NULL,
                email TEXT NOT NULL
            );
        , parameters: None
[2023-07-22T05:54:12.152+0000] {postgres.py:96} INFO - NOTICE:  relation "users" already exists, skipping

[2023-07-22T05:54:12.179+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=create_table, execution_date=20230713T000000, start_date=20230722T055410, end_date=20230722T055412
[2023-07-22T05:54:12.226+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-07-22T05:54:12.421+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-07-22T05:56:54.034+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table scheduled__2023-07-13T00:00:00+00:00 [queued]>
[2023-07-22T05:56:54.121+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table scheduled__2023-07-13T00:00:00+00:00 [queued]>
[2023-07-22T05:56:54.132+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-07-22T05:56:54.138+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-07-22T05:56:54.138+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-07-22T05:56:54.313+0000] {taskinstance.py:1383} INFO - Executing <Task(PostgresOperator): create_table> on 2023-07-13 00:00:00+00:00
[2023-07-22T05:56:54.349+0000] {standard_task_runner.py:55} INFO - Started process 5448 to run task
[2023-07-22T05:56:54.372+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'create_table', 'scheduled__2023-07-13T00:00:00+00:00', '--job-id', '98', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmproxelfuf']
[2023-07-22T05:56:54.393+0000] {standard_task_runner.py:83} INFO - Job 98: Subtask create_table
[2023-07-22T05:56:54.737+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing.create_table scheduled__2023-07-13T00:00:00+00:00 [running]> on host ef71d3f52879
[2023-07-22T05:56:54.892+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2023-07-13T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-07-13T00:00:00+00:00
[2023-07-22T05:56:54.926+0000] {base.py:71} INFO - Using connection ID 'postgres' for task execution.
[2023-07-22T05:56:54.936+0000] {sql.py:315} INFO - Running statement: 
            CREATE TABLE IF NOT EXISTS users (
                firstname TEXT NOT NULL,
                lastname TEXT NOT NULL,
                country TEXT NOT NULL,
                username TEXT NOT NULL,
                password TEXT NOT NULL,
                email TEXT NOT NULL
            );
        , parameters: None
[2023-07-22T05:56:54.940+0000] {postgres.py:96} INFO - NOTICE:  relation "users" already exists, skipping

[2023-07-22T05:56:54.955+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=create_table, execution_date=20230713T000000, start_date=20230722T055654, end_date=20230722T055654
[2023-07-22T05:56:55.010+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-07-22T05:56:55.081+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
